Demos/Blazorise.Demo/Pages/Tests/DataGridPage.razor

```diff
@@ -319,7 +319,7 @@
                                 }
                             </DisplayTemplate>
                             <EditTemplate>
-                                <Select TValue="string" SelectedValue="@((string)( context.CellValue ))" SelectedValueChanged="@(( v ) => { context.CellValue = v; context.UpdateCell( nameof( Employee.FirstName ), "Mr" +(v=="F"?"s":"") + context.ReadCell(nameof(Employee.FirstName)) ); })">
+                                <Select TValue="string" SelectedValue="@((string)( context.CellValue ))" SelectedValueChanged="@(( v ) => { context.CellValue = v; context.UpdateCell( nameof( Employee.FirstName ), TitleFromGender( v ) + context.ReadCell( nameof( Employee.FirstName ) ) ); })">

```

Comment:
Is it still missing the space? haha? Meaning it will do something like: 
`Mr.Jonh` instead of `Mr. John`?

Reply:
Done